examples/typescript/azure-app-service/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "azure-app-service-docker",
```

Comment:
please name it `@examples/azure-app-service-docker` - see https://github.com/hashicorp/terraform-cdk/blob/7faebbe7ace248377c3ca35c7d40697e797f14e2/CONTRIBUTING.md#examples

Reply:
No problem !!! Sorry my mistake 😊

Reply:
No worries, that's something which got introduced very recently